From 7ef0c57ae7c852bb8f35310b0bd5708bb2a98c4a Mon Sep 17 00:00:00 2001
From: Josua Mayer <josua@solid-run.com>
Date: Tue, 8 Oct 2024 12:30:42 +0200
Subject: [PATCH 8/9] lx2160acex7: change 2.2GHz configuration platform clock
 to 750MHz

2.2GHz cpu clock is only for accordingly binned part number from NXP.
These parts support platform clock up to 750MHz.

Update the configuration accordingly to 2200_750_xxxx and drop the
combination with 2200_700.

Signed-off-by: Josua Mayer <josua@solid-run.com>
---
 ...2400_8_5_2_auto.rcw => rcw_2200_750_2400_8_5_2_auto.rcw} | 2 +-
 ...2400_8_5_2_sdhc.rcw => rcw_2200_750_2400_8_5_2_sdhc.rcw} | 2 +-
 ...2400_8_5_2_xspi.rcw => rcw_2200_750_2400_8_5_2_xspi.rcw} | 2 +-
 ...2600_8_5_2_auto.rcw => rcw_2200_750_2600_8_5_2_auto.rcw} | 2 +-
 ...2600_8_5_2_sdhc.rcw => rcw_2200_750_2600_8_5_2_sdhc.rcw} | 2 +-
 ...2600_8_5_2_xspi.rcw => rcw_2200_750_2600_8_5_2_xspi.rcw} | 2 +-
 ...2900_8_5_2_auto.rcw => rcw_2200_750_2900_8_5_2_auto.rcw} | 2 +-
 ...2900_8_5_2_sdhc.rcw => rcw_2200_750_2900_8_5_2_sdhc.rcw} | 2 +-
 ...2900_8_5_2_xspi.rcw => rcw_2200_750_2900_8_5_2_xspi.rcw} | 2 +-
 ...3200_8_5_2_auto.rcw => rcw_2200_750_3200_8_5_2_auto.rcw} | 2 +-
 ...3200_8_5_2_sdhc.rcw => rcw_2200_750_3200_8_5_2_sdhc.rcw} | 2 +-
 ...3200_8_5_2_xspi.rcw => rcw_2200_750_3200_8_5_2_xspi.rcw} | 2 +-
 .../{pll_2200_700_xxxx.rcwi => pll_2200_750_xxxx.rcwi}      | 6 +++---
 ...2400_8_5_2_auto.rcw => rcw_2200_750_2400_8_5_2_auto.rcw} | 2 +-
 ...2400_8_5_2_sdhc.rcw => rcw_2200_750_2400_8_5_2_sdhc.rcw} | 2 +-
 ...2400_8_5_2_xspi.rcw => rcw_2200_750_2400_8_5_2_xspi.rcw} | 2 +-
 ...2600_8_5_2_auto.rcw => rcw_2200_750_2600_8_5_2_auto.rcw} | 2 +-
 ...2600_8_5_2_sdhc.rcw => rcw_2200_750_2600_8_5_2_sdhc.rcw} | 2 +-
 ...2600_8_5_2_xspi.rcw => rcw_2200_750_2600_8_5_2_xspi.rcw} | 2 +-
 ...2900_8_5_2_auto.rcw => rcw_2200_750_2900_8_5_2_auto.rcw} | 2 +-
 ...2900_8_5_2_sdhc.rcw => rcw_2200_750_2900_8_5_2_sdhc.rcw} | 2 +-
 ...2900_8_5_2_xspi.rcw => rcw_2200_750_2900_8_5_2_xspi.rcw} | 2 +-
 ...3200_8_5_2_auto.rcw => rcw_2200_750_3200_8_5_2_auto.rcw} | 2 +-
 ...3200_8_5_2_sdhc.rcw => rcw_2200_750_3200_8_5_2_sdhc.rcw} | 2 +-
 ...3200_8_5_2_xspi.rcw => rcw_2200_750_3200_8_5_2_xspi.rcw} | 2 +-
 25 files changed, 27 insertions(+), 27 deletions(-)
 rename lx2160acex7/clearfog-cx/{rcw_2200_700_2400_8_5_2_auto.rcw => rcw_2200_750_2400_8_5_2_auto.rcw} (91%)
 rename lx2160acex7/clearfog-cx/{rcw_2200_700_2400_8_5_2_sdhc.rcw => rcw_2200_750_2400_8_5_2_sdhc.rcw} (91%)
 rename lx2160acex7/clearfog-cx/{rcw_2200_700_2400_8_5_2_xspi.rcw => rcw_2200_750_2400_8_5_2_xspi.rcw} (91%)
 rename lx2160acex7/clearfog-cx/{rcw_2200_700_2600_8_5_2_auto.rcw => rcw_2200_750_2600_8_5_2_auto.rcw} (91%)
 rename lx2160acex7/clearfog-cx/{rcw_2200_700_2600_8_5_2_sdhc.rcw => rcw_2200_750_2600_8_5_2_sdhc.rcw} (91%)
 rename lx2160acex7/clearfog-cx/{rcw_2200_700_2600_8_5_2_xspi.rcw => rcw_2200_750_2600_8_5_2_xspi.rcw} (91%)
 rename lx2160acex7/clearfog-cx/{rcw_2200_700_2900_8_5_2_auto.rcw => rcw_2200_750_2900_8_5_2_auto.rcw} (91%)
 rename lx2160acex7/clearfog-cx/{rcw_2200_700_2900_8_5_2_sdhc.rcw => rcw_2200_750_2900_8_5_2_sdhc.rcw} (91%)
 rename lx2160acex7/clearfog-cx/{rcw_2200_700_2900_8_5_2_xspi.rcw => rcw_2200_750_2900_8_5_2_xspi.rcw} (91%)
 rename lx2160acex7/clearfog-cx/{rcw_2200_700_3200_8_5_2_auto.rcw => rcw_2200_750_3200_8_5_2_auto.rcw} (91%)
 rename lx2160acex7/clearfog-cx/{rcw_2200_700_3200_8_5_2_sdhc.rcw => rcw_2200_750_3200_8_5_2_sdhc.rcw} (91%)
 rename lx2160acex7/clearfog-cx/{rcw_2200_700_3200_8_5_2_xspi.rcw => rcw_2200_750_3200_8_5_2_xspi.rcw} (91%)
 rename lx2160acex7/include/{pll_2200_700_xxxx.rcwi => pll_2200_750_xxxx.rcwi} (61%)
 rename lx2160acex7_rev2/clearfog-cx/{rcw_2200_700_2400_8_5_2_auto.rcw => rcw_2200_750_2400_8_5_2_auto.rcw} (91%)
 rename lx2160acex7_rev2/clearfog-cx/{rcw_2200_700_2400_8_5_2_sdhc.rcw => rcw_2200_750_2400_8_5_2_sdhc.rcw} (91%)
 rename lx2160acex7_rev2/clearfog-cx/{rcw_2200_700_2400_8_5_2_xspi.rcw => rcw_2200_750_2400_8_5_2_xspi.rcw} (91%)
 rename lx2160acex7_rev2/clearfog-cx/{rcw_2200_700_2600_8_5_2_auto.rcw => rcw_2200_750_2600_8_5_2_auto.rcw} (91%)
 rename lx2160acex7_rev2/clearfog-cx/{rcw_2200_700_2600_8_5_2_sdhc.rcw => rcw_2200_750_2600_8_5_2_sdhc.rcw} (91%)
 rename lx2160acex7_rev2/clearfog-cx/{rcw_2200_700_2600_8_5_2_xspi.rcw => rcw_2200_750_2600_8_5_2_xspi.rcw} (91%)
 rename lx2160acex7_rev2/clearfog-cx/{rcw_2200_700_2900_8_5_2_auto.rcw => rcw_2200_750_2900_8_5_2_auto.rcw} (91%)
 rename lx2160acex7_rev2/clearfog-cx/{rcw_2200_700_2900_8_5_2_sdhc.rcw => rcw_2200_750_2900_8_5_2_sdhc.rcw} (91%)
 rename lx2160acex7_rev2/clearfog-cx/{rcw_2200_700_2900_8_5_2_xspi.rcw => rcw_2200_750_2900_8_5_2_xspi.rcw} (91%)
 rename lx2160acex7_rev2/clearfog-cx/{rcw_2200_700_3200_8_5_2_auto.rcw => rcw_2200_750_3200_8_5_2_auto.rcw} (91%)
 rename lx2160acex7_rev2/clearfog-cx/{rcw_2200_700_3200_8_5_2_sdhc.rcw => rcw_2200_750_3200_8_5_2_sdhc.rcw} (91%)
 rename lx2160acex7_rev2/clearfog-cx/{rcw_2200_700_3200_8_5_2_xspi.rcw => rcw_2200_750_3200_8_5_2_xspi.rcw} (91%)

diff --git a/lx2160acex7/clearfog-cx/rcw_2200_700_2400_8_5_2_auto.rcw b/lx2160acex7/clearfog-cx/rcw_2200_750_2400_8_5_2_auto.rcw
similarity index 91%
rename from lx2160acex7/clearfog-cx/rcw_2200_700_2400_8_5_2_auto.rcw
rename to lx2160acex7/clearfog-cx/rcw_2200_750_2400_8_5_2_auto.rcw
index 464a285..533fba1 100644
--- a/lx2160acex7/clearfog-cx/rcw_2200_700_2400_8_5_2_auto.rcw
+++ b/lx2160acex7/clearfog-cx/rcw_2200_750_2400_8_5_2_auto.rcw
@@ -13,7 +13,7 @@
 
 #define LX_SR 1
 #include <../lx2160asi/lx2160a.rcwi>
-#include <../lx2160acex7/include/pll_2200_700_xxxx.rcwi>
+#include <../lx2160acex7/include/pll_2200_750_xxxx.rcwi>
 #include <../lx2160acex7/include/pll_xxxx_xxx_2400.rcwi>
 #include <../lx2160acex7/include/common.rcwi>
 #include <../lx2160acex7/include/SD1_8.rcwi>
diff --git a/lx2160acex7/clearfog-cx/rcw_2200_700_2400_8_5_2_sdhc.rcw b/lx2160acex7/clearfog-cx/rcw_2200_750_2400_8_5_2_sdhc.rcw
similarity index 91%
rename from lx2160acex7/clearfog-cx/rcw_2200_700_2400_8_5_2_sdhc.rcw
rename to lx2160acex7/clearfog-cx/rcw_2200_750_2400_8_5_2_sdhc.rcw
index 6f696f5..845ab35 100644
--- a/lx2160acex7/clearfog-cx/rcw_2200_700_2400_8_5_2_sdhc.rcw
+++ b/lx2160acex7/clearfog-cx/rcw_2200_750_2400_8_5_2_sdhc.rcw
@@ -15,7 +15,7 @@
 #define LX_SR 1
 #define LX_BOOTSOURCE_SDHC
 #include <../lx2160asi/lx2160a.rcwi>
-#include <../lx2160acex7/include/pll_2200_700_xxxx.rcwi>
+#include <../lx2160acex7/include/pll_2200_750_xxxx.rcwi>
 #include <../lx2160acex7/include/pll_xxxx_xxx_2400.rcwi>
 #include <../lx2160acex7/include/common.rcwi>
 #include <../lx2160acex7/include/SD1_8.rcwi>
diff --git a/lx2160acex7/clearfog-cx/rcw_2200_700_2400_8_5_2_xspi.rcw b/lx2160acex7/clearfog-cx/rcw_2200_750_2400_8_5_2_xspi.rcw
similarity index 91%
rename from lx2160acex7/clearfog-cx/rcw_2200_700_2400_8_5_2_xspi.rcw
rename to lx2160acex7/clearfog-cx/rcw_2200_750_2400_8_5_2_xspi.rcw
index f23be28..765758e 100644
--- a/lx2160acex7/clearfog-cx/rcw_2200_700_2400_8_5_2_xspi.rcw
+++ b/lx2160acex7/clearfog-cx/rcw_2200_750_2400_8_5_2_xspi.rcw
@@ -15,7 +15,7 @@
 #define LX_SR 1
 #define LX_BOOTSOURCE_XSPI
 #include <../lx2160asi/lx2160a.rcwi>
-#include <../lx2160acex7/include/pll_2200_700_xxxx.rcwi>
+#include <../lx2160acex7/include/pll_2200_750_xxxx.rcwi>
 #include <../lx2160acex7/include/pll_xxxx_xxx_2400.rcwi>
 #include <../lx2160acex7/include/common.rcwi>
 #include <../lx2160acex7/include/SD1_8.rcwi>
diff --git a/lx2160acex7/clearfog-cx/rcw_2200_700_2600_8_5_2_auto.rcw b/lx2160acex7/clearfog-cx/rcw_2200_750_2600_8_5_2_auto.rcw
similarity index 91%
rename from lx2160acex7/clearfog-cx/rcw_2200_700_2600_8_5_2_auto.rcw
rename to lx2160acex7/clearfog-cx/rcw_2200_750_2600_8_5_2_auto.rcw
index 1ef7db6..c09807c 100644
--- a/lx2160acex7/clearfog-cx/rcw_2200_700_2600_8_5_2_auto.rcw
+++ b/lx2160acex7/clearfog-cx/rcw_2200_750_2600_8_5_2_auto.rcw
@@ -13,7 +13,7 @@
 
 #define LX_SR 1
 #include <../lx2160asi/lx2160a.rcwi>
-#include <../lx2160acex7/include/pll_2200_700_xxxx.rcwi>
+#include <../lx2160acex7/include/pll_2200_750_xxxx.rcwi>
 #include <../lx2160acex7/include/pll_xxxx_xxx_2600.rcwi>
 #include <../lx2160acex7/include/common.rcwi>
 #include <../lx2160acex7/include/SD1_8.rcwi>
diff --git a/lx2160acex7/clearfog-cx/rcw_2200_700_2600_8_5_2_sdhc.rcw b/lx2160acex7/clearfog-cx/rcw_2200_750_2600_8_5_2_sdhc.rcw
similarity index 91%
rename from lx2160acex7/clearfog-cx/rcw_2200_700_2600_8_5_2_sdhc.rcw
rename to lx2160acex7/clearfog-cx/rcw_2200_750_2600_8_5_2_sdhc.rcw
index 46d84cb..7808b12 100644
--- a/lx2160acex7/clearfog-cx/rcw_2200_700_2600_8_5_2_sdhc.rcw
+++ b/lx2160acex7/clearfog-cx/rcw_2200_750_2600_8_5_2_sdhc.rcw
@@ -15,7 +15,7 @@
 #define LX_SR 1
 #define LX_BOOTSOURCE_SDHC
 #include <../lx2160asi/lx2160a.rcwi>
-#include <../lx2160acex7/include/pll_2200_700_xxxx.rcwi>
+#include <../lx2160acex7/include/pll_2200_750_xxxx.rcwi>
 #include <../lx2160acex7/include/pll_xxxx_xxx_2600.rcwi>
 #include <../lx2160acex7/include/common.rcwi>
 #include <../lx2160acex7/include/SD1_8.rcwi>
diff --git a/lx2160acex7/clearfog-cx/rcw_2200_700_2600_8_5_2_xspi.rcw b/lx2160acex7/clearfog-cx/rcw_2200_750_2600_8_5_2_xspi.rcw
similarity index 91%
rename from lx2160acex7/clearfog-cx/rcw_2200_700_2600_8_5_2_xspi.rcw
rename to lx2160acex7/clearfog-cx/rcw_2200_750_2600_8_5_2_xspi.rcw
index 62cf89e..33bef8b 100644
--- a/lx2160acex7/clearfog-cx/rcw_2200_700_2600_8_5_2_xspi.rcw
+++ b/lx2160acex7/clearfog-cx/rcw_2200_750_2600_8_5_2_xspi.rcw
@@ -15,7 +15,7 @@
 #define LX_SR 1
 #define LX_BOOTSOURCE_XSPI
 #include <../lx2160asi/lx2160a.rcwi>
-#include <../lx2160acex7/include/pll_2200_700_xxxx.rcwi>
+#include <../lx2160acex7/include/pll_2200_750_xxxx.rcwi>
 #include <../lx2160acex7/include/pll_xxxx_xxx_2600.rcwi>
 #include <../lx2160acex7/include/common.rcwi>
 #include <../lx2160acex7/include/SD1_8.rcwi>
diff --git a/lx2160acex7/clearfog-cx/rcw_2200_700_2900_8_5_2_auto.rcw b/lx2160acex7/clearfog-cx/rcw_2200_750_2900_8_5_2_auto.rcw
similarity index 91%
rename from lx2160acex7/clearfog-cx/rcw_2200_700_2900_8_5_2_auto.rcw
rename to lx2160acex7/clearfog-cx/rcw_2200_750_2900_8_5_2_auto.rcw
index d021306..0f3d8a3 100644
--- a/lx2160acex7/clearfog-cx/rcw_2200_700_2900_8_5_2_auto.rcw
+++ b/lx2160acex7/clearfog-cx/rcw_2200_750_2900_8_5_2_auto.rcw
@@ -13,7 +13,7 @@
 
 #define LX_SR 1
 #include <../lx2160asi/lx2160a.rcwi>
-#include <../lx2160acex7/include/pll_2200_700_xxxx.rcwi>
+#include <../lx2160acex7/include/pll_2200_750_xxxx.rcwi>
 #include <../lx2160acex7/include/pll_xxxx_xxx_2900.rcwi>
 #include <../lx2160acex7/include/common.rcwi>
 #include <../lx2160acex7/include/SD1_8.rcwi>
diff --git a/lx2160acex7/clearfog-cx/rcw_2200_700_2900_8_5_2_sdhc.rcw b/lx2160acex7/clearfog-cx/rcw_2200_750_2900_8_5_2_sdhc.rcw
similarity index 91%
rename from lx2160acex7/clearfog-cx/rcw_2200_700_2900_8_5_2_sdhc.rcw
rename to lx2160acex7/clearfog-cx/rcw_2200_750_2900_8_5_2_sdhc.rcw
index 55e6b2b..68452e0 100644
--- a/lx2160acex7/clearfog-cx/rcw_2200_700_2900_8_5_2_sdhc.rcw
+++ b/lx2160acex7/clearfog-cx/rcw_2200_750_2900_8_5_2_sdhc.rcw
@@ -15,7 +15,7 @@
 #define LX_SR 1
 #define LX_BOOTSOURCE_SDHC
 #include <../lx2160asi/lx2160a.rcwi>
-#include <../lx2160acex7/include/pll_2200_700_xxxx.rcwi>
+#include <../lx2160acex7/include/pll_2200_750_xxxx.rcwi>
 #include <../lx2160acex7/include/pll_xxxx_xxx_2900.rcwi>
 #include <../lx2160acex7/include/common.rcwi>
 #include <../lx2160acex7/include/SD1_8.rcwi>
diff --git a/lx2160acex7/clearfog-cx/rcw_2200_700_2900_8_5_2_xspi.rcw b/lx2160acex7/clearfog-cx/rcw_2200_750_2900_8_5_2_xspi.rcw
similarity index 91%
rename from lx2160acex7/clearfog-cx/rcw_2200_700_2900_8_5_2_xspi.rcw
rename to lx2160acex7/clearfog-cx/rcw_2200_750_2900_8_5_2_xspi.rcw
index 13ec066..0069109 100644
--- a/lx2160acex7/clearfog-cx/rcw_2200_700_2900_8_5_2_xspi.rcw
+++ b/lx2160acex7/clearfog-cx/rcw_2200_750_2900_8_5_2_xspi.rcw
@@ -15,7 +15,7 @@
 #define LX_SR 1
 #define LX_BOOTSOURCE_XSPI
 #include <../lx2160asi/lx2160a.rcwi>
-#include <../lx2160acex7/include/pll_2200_700_xxxx.rcwi>
+#include <../lx2160acex7/include/pll_2200_750_xxxx.rcwi>
 #include <../lx2160acex7/include/pll_xxxx_xxx_2900.rcwi>
 #include <../lx2160acex7/include/common.rcwi>
 #include <../lx2160acex7/include/SD1_8.rcwi>
diff --git a/lx2160acex7/clearfog-cx/rcw_2200_700_3200_8_5_2_auto.rcw b/lx2160acex7/clearfog-cx/rcw_2200_750_3200_8_5_2_auto.rcw
similarity index 91%
rename from lx2160acex7/clearfog-cx/rcw_2200_700_3200_8_5_2_auto.rcw
rename to lx2160acex7/clearfog-cx/rcw_2200_750_3200_8_5_2_auto.rcw
index bfef1d4..aa2fb4b 100644
--- a/lx2160acex7/clearfog-cx/rcw_2200_700_3200_8_5_2_auto.rcw
+++ b/lx2160acex7/clearfog-cx/rcw_2200_750_3200_8_5_2_auto.rcw
@@ -13,7 +13,7 @@
 
 #define LX_SR 1
 #include <../lx2160asi/lx2160a.rcwi>
-#include <../lx2160acex7/include/pll_2200_700_xxxx.rcwi>
+#include <../lx2160acex7/include/pll_2200_750_xxxx.rcwi>
 #include <../lx2160acex7/include/pll_xxxx_xxx_3200.rcwi>
 #include <../lx2160acex7/include/common.rcwi>
 #include <../lx2160acex7/include/SD1_8.rcwi>
diff --git a/lx2160acex7/clearfog-cx/rcw_2200_700_3200_8_5_2_sdhc.rcw b/lx2160acex7/clearfog-cx/rcw_2200_750_3200_8_5_2_sdhc.rcw
similarity index 91%
rename from lx2160acex7/clearfog-cx/rcw_2200_700_3200_8_5_2_sdhc.rcw
rename to lx2160acex7/clearfog-cx/rcw_2200_750_3200_8_5_2_sdhc.rcw
index 2978d72..6f06730 100644
--- a/lx2160acex7/clearfog-cx/rcw_2200_700_3200_8_5_2_sdhc.rcw
+++ b/lx2160acex7/clearfog-cx/rcw_2200_750_3200_8_5_2_sdhc.rcw
@@ -15,7 +15,7 @@
 #define LX_SR 1
 #define LX_BOOTSOURCE_SDHC
 #include <../lx2160asi/lx2160a.rcwi>
-#include <../lx2160acex7/include/pll_2200_700_xxxx.rcwi>
+#include <../lx2160acex7/include/pll_2200_750_xxxx.rcwi>
 #include <../lx2160acex7/include/pll_xxxx_xxx_3200.rcwi>
 #include <../lx2160acex7/include/common.rcwi>
 #include <../lx2160acex7/include/SD1_8.rcwi>
diff --git a/lx2160acex7/clearfog-cx/rcw_2200_700_3200_8_5_2_xspi.rcw b/lx2160acex7/clearfog-cx/rcw_2200_750_3200_8_5_2_xspi.rcw
similarity index 91%
rename from lx2160acex7/clearfog-cx/rcw_2200_700_3200_8_5_2_xspi.rcw
rename to lx2160acex7/clearfog-cx/rcw_2200_750_3200_8_5_2_xspi.rcw
index b80f8cb..94dcc9b 100644
--- a/lx2160acex7/clearfog-cx/rcw_2200_700_3200_8_5_2_xspi.rcw
+++ b/lx2160acex7/clearfog-cx/rcw_2200_750_3200_8_5_2_xspi.rcw
@@ -15,7 +15,7 @@
 #define LX_SR 1
 #define LX_BOOTSOURCE_XSPI
 #include <../lx2160asi/lx2160a.rcwi>
-#include <../lx2160acex7/include/pll_2200_700_xxxx.rcwi>
+#include <../lx2160acex7/include/pll_2200_750_xxxx.rcwi>
 #include <../lx2160acex7/include/pll_xxxx_xxx_3200.rcwi>
 #include <../lx2160acex7/include/common.rcwi>
 #include <../lx2160acex7/include/SD1_8.rcwi>
diff --git a/lx2160acex7/include/pll_2200_700_xxxx.rcwi b/lx2160acex7/include/pll_2200_750_xxxx.rcwi
similarity index 61%
rename from lx2160acex7/include/pll_2200_700_xxxx.rcwi
rename to lx2160acex7/include/pll_2200_750_xxxx.rcwi
index 91d1a9b..0f57b67 100644
--- a/lx2160acex7/include/pll_2200_700_xxxx.rcwi
+++ b/lx2160acex7/include/pll_2200_750_xxxx.rcwi
@@ -1,11 +1,11 @@
 /*
  * Core and Platform Clocks:
- * - Platform: 700MHz
+ * - Platform: 750MHz
  * - Core: 2200MHz
  */
 
-/* platform clock is system clock mul 14 div 2 = 700 */
-SYS_PLL_RAT=14
+/* platform clock is system clock mul 15 div 2 = 750 */
+SYS_PLL_RAT=15
 
 /* core clocks are 2200 */
 CGA_PLL1_RAT=22
diff --git a/lx2160acex7_rev2/clearfog-cx/rcw_2200_700_2400_8_5_2_auto.rcw b/lx2160acex7_rev2/clearfog-cx/rcw_2200_750_2400_8_5_2_auto.rcw
similarity index 91%
rename from lx2160acex7_rev2/clearfog-cx/rcw_2200_700_2400_8_5_2_auto.rcw
rename to lx2160acex7_rev2/clearfog-cx/rcw_2200_750_2400_8_5_2_auto.rcw
index d59ceed..4185ea6 100644
--- a/lx2160acex7_rev2/clearfog-cx/rcw_2200_700_2400_8_5_2_auto.rcw
+++ b/lx2160acex7_rev2/clearfog-cx/rcw_2200_750_2400_8_5_2_auto.rcw
@@ -13,7 +13,7 @@
 
 #define LX_SR 2
 #include <../lx2160asi/lx2160a.rcwi>
-#include <../lx2160acex7/include/pll_2200_700_xxxx.rcwi>
+#include <../lx2160acex7/include/pll_2200_750_xxxx.rcwi>
 #include <../lx2160acex7/include/pll_xxxx_xxx_2400.rcwi>
 #include <../lx2160acex7/include/common.rcwi>
 #include <../lx2160acex7/include/SD1_8.rcwi>
diff --git a/lx2160acex7_rev2/clearfog-cx/rcw_2200_700_2400_8_5_2_sdhc.rcw b/lx2160acex7_rev2/clearfog-cx/rcw_2200_750_2400_8_5_2_sdhc.rcw
similarity index 91%
rename from lx2160acex7_rev2/clearfog-cx/rcw_2200_700_2400_8_5_2_sdhc.rcw
rename to lx2160acex7_rev2/clearfog-cx/rcw_2200_750_2400_8_5_2_sdhc.rcw
index 20f7d58..93b10d2 100644
--- a/lx2160acex7_rev2/clearfog-cx/rcw_2200_700_2400_8_5_2_sdhc.rcw
+++ b/lx2160acex7_rev2/clearfog-cx/rcw_2200_750_2400_8_5_2_sdhc.rcw
@@ -15,7 +15,7 @@
 #define LX_SR 2
 #define LX_BOOTSOURCE_SDHC
 #include <../lx2160asi/lx2160a.rcwi>
-#include <../lx2160acex7/include/pll_2200_700_xxxx.rcwi>
+#include <../lx2160acex7/include/pll_2200_750_xxxx.rcwi>
 #include <../lx2160acex7/include/pll_xxxx_xxx_2400.rcwi>
 #include <../lx2160acex7/include/common.rcwi>
 #include <../lx2160acex7/include/SD1_8.rcwi>
diff --git a/lx2160acex7_rev2/clearfog-cx/rcw_2200_700_2400_8_5_2_xspi.rcw b/lx2160acex7_rev2/clearfog-cx/rcw_2200_750_2400_8_5_2_xspi.rcw
similarity index 91%
rename from lx2160acex7_rev2/clearfog-cx/rcw_2200_700_2400_8_5_2_xspi.rcw
rename to lx2160acex7_rev2/clearfog-cx/rcw_2200_750_2400_8_5_2_xspi.rcw
index 9643b52..feb3e42 100644
--- a/lx2160acex7_rev2/clearfog-cx/rcw_2200_700_2400_8_5_2_xspi.rcw
+++ b/lx2160acex7_rev2/clearfog-cx/rcw_2200_750_2400_8_5_2_xspi.rcw
@@ -15,7 +15,7 @@
 #define LX_SR 2
 #define LX_BOOTSOURCE_XSPI
 #include <../lx2160asi/lx2160a.rcwi>
-#include <../lx2160acex7/include/pll_2200_700_xxxx.rcwi>
+#include <../lx2160acex7/include/pll_2200_750_xxxx.rcwi>
 #include <../lx2160acex7/include/pll_xxxx_xxx_2400.rcwi>
 #include <../lx2160acex7/include/common.rcwi>
 #include <../lx2160acex7/include/SD1_8.rcwi>
diff --git a/lx2160acex7_rev2/clearfog-cx/rcw_2200_700_2600_8_5_2_auto.rcw b/lx2160acex7_rev2/clearfog-cx/rcw_2200_750_2600_8_5_2_auto.rcw
similarity index 91%
rename from lx2160acex7_rev2/clearfog-cx/rcw_2200_700_2600_8_5_2_auto.rcw
rename to lx2160acex7_rev2/clearfog-cx/rcw_2200_750_2600_8_5_2_auto.rcw
index 1bf815b..e04fcfb 100644
--- a/lx2160acex7_rev2/clearfog-cx/rcw_2200_700_2600_8_5_2_auto.rcw
+++ b/lx2160acex7_rev2/clearfog-cx/rcw_2200_750_2600_8_5_2_auto.rcw
@@ -13,7 +13,7 @@
 
 #define LX_SR 2
 #include <../lx2160asi/lx2160a.rcwi>
-#include <../lx2160acex7/include/pll_2200_700_xxxx.rcwi>
+#include <../lx2160acex7/include/pll_2200_750_xxxx.rcwi>
 #include <../lx2160acex7/include/pll_xxxx_xxx_2600.rcwi>
 #include <../lx2160acex7/include/common.rcwi>
 #include <../lx2160acex7/include/SD1_8.rcwi>
diff --git a/lx2160acex7_rev2/clearfog-cx/rcw_2200_700_2600_8_5_2_sdhc.rcw b/lx2160acex7_rev2/clearfog-cx/rcw_2200_750_2600_8_5_2_sdhc.rcw
similarity index 91%
rename from lx2160acex7_rev2/clearfog-cx/rcw_2200_700_2600_8_5_2_sdhc.rcw
rename to lx2160acex7_rev2/clearfog-cx/rcw_2200_750_2600_8_5_2_sdhc.rcw
index 0f93cf2..32225fe 100644
--- a/lx2160acex7_rev2/clearfog-cx/rcw_2200_700_2600_8_5_2_sdhc.rcw
+++ b/lx2160acex7_rev2/clearfog-cx/rcw_2200_750_2600_8_5_2_sdhc.rcw
@@ -15,7 +15,7 @@
 #define LX_SR 2
 #define LX_BOOTSOURCE_SDHC
 #include <../lx2160asi/lx2160a.rcwi>
-#include <../lx2160acex7/include/pll_2200_700_xxxx.rcwi>
+#include <../lx2160acex7/include/pll_2200_750_xxxx.rcwi>
 #include <../lx2160acex7/include/pll_xxxx_xxx_2600.rcwi>
 #include <../lx2160acex7/include/common.rcwi>
 #include <../lx2160acex7/include/SD1_8.rcwi>
diff --git a/lx2160acex7_rev2/clearfog-cx/rcw_2200_700_2600_8_5_2_xspi.rcw b/lx2160acex7_rev2/clearfog-cx/rcw_2200_750_2600_8_5_2_xspi.rcw
similarity index 91%
rename from lx2160acex7_rev2/clearfog-cx/rcw_2200_700_2600_8_5_2_xspi.rcw
rename to lx2160acex7_rev2/clearfog-cx/rcw_2200_750_2600_8_5_2_xspi.rcw
index 79cea0d..6fbba40 100644
--- a/lx2160acex7_rev2/clearfog-cx/rcw_2200_700_2600_8_5_2_xspi.rcw
+++ b/lx2160acex7_rev2/clearfog-cx/rcw_2200_750_2600_8_5_2_xspi.rcw
@@ -15,7 +15,7 @@
 #define LX_SR 2
 #define LX_BOOTSOURCE_XSPI
 #include <../lx2160asi/lx2160a.rcwi>
-#include <../lx2160acex7/include/pll_2200_700_xxxx.rcwi>
+#include <../lx2160acex7/include/pll_2200_750_xxxx.rcwi>
 #include <../lx2160acex7/include/pll_xxxx_xxx_2600.rcwi>
 #include <../lx2160acex7/include/common.rcwi>
 #include <../lx2160acex7/include/SD1_8.rcwi>
diff --git a/lx2160acex7_rev2/clearfog-cx/rcw_2200_700_2900_8_5_2_auto.rcw b/lx2160acex7_rev2/clearfog-cx/rcw_2200_750_2900_8_5_2_auto.rcw
similarity index 91%
rename from lx2160acex7_rev2/clearfog-cx/rcw_2200_700_2900_8_5_2_auto.rcw
rename to lx2160acex7_rev2/clearfog-cx/rcw_2200_750_2900_8_5_2_auto.rcw
index 01138dc..b9797ca 100644
--- a/lx2160acex7_rev2/clearfog-cx/rcw_2200_700_2900_8_5_2_auto.rcw
+++ b/lx2160acex7_rev2/clearfog-cx/rcw_2200_750_2900_8_5_2_auto.rcw
@@ -13,7 +13,7 @@
 
 #define LX_SR 2
 #include <../lx2160asi/lx2160a.rcwi>
-#include <../lx2160acex7/include/pll_2200_700_xxxx.rcwi>
+#include <../lx2160acex7/include/pll_2200_750_xxxx.rcwi>
 #include <../lx2160acex7/include/pll_xxxx_xxx_2900.rcwi>
 #include <../lx2160acex7/include/common.rcwi>
 #include <../lx2160acex7/include/SD1_8.rcwi>
diff --git a/lx2160acex7_rev2/clearfog-cx/rcw_2200_700_2900_8_5_2_sdhc.rcw b/lx2160acex7_rev2/clearfog-cx/rcw_2200_750_2900_8_5_2_sdhc.rcw
similarity index 91%
rename from lx2160acex7_rev2/clearfog-cx/rcw_2200_700_2900_8_5_2_sdhc.rcw
rename to lx2160acex7_rev2/clearfog-cx/rcw_2200_750_2900_8_5_2_sdhc.rcw
index a06759e..a7b5bd2 100644
--- a/lx2160acex7_rev2/clearfog-cx/rcw_2200_700_2900_8_5_2_sdhc.rcw
+++ b/lx2160acex7_rev2/clearfog-cx/rcw_2200_750_2900_8_5_2_sdhc.rcw
@@ -15,7 +15,7 @@
 #define LX_SR 2
 #define LX_BOOTSOURCE_SDHC
 #include <../lx2160asi/lx2160a.rcwi>
-#include <../lx2160acex7/include/pll_2200_700_xxxx.rcwi>
+#include <../lx2160acex7/include/pll_2200_750_xxxx.rcwi>
 #include <../lx2160acex7/include/pll_xxxx_xxx_2900.rcwi>
 #include <../lx2160acex7/include/common.rcwi>
 #include <../lx2160acex7/include/SD1_8.rcwi>
diff --git a/lx2160acex7_rev2/clearfog-cx/rcw_2200_700_2900_8_5_2_xspi.rcw b/lx2160acex7_rev2/clearfog-cx/rcw_2200_750_2900_8_5_2_xspi.rcw
similarity index 91%
rename from lx2160acex7_rev2/clearfog-cx/rcw_2200_700_2900_8_5_2_xspi.rcw
rename to lx2160acex7_rev2/clearfog-cx/rcw_2200_750_2900_8_5_2_xspi.rcw
index b12898a..273d91c 100644
--- a/lx2160acex7_rev2/clearfog-cx/rcw_2200_700_2900_8_5_2_xspi.rcw
+++ b/lx2160acex7_rev2/clearfog-cx/rcw_2200_750_2900_8_5_2_xspi.rcw
@@ -15,7 +15,7 @@
 #define LX_SR 2
 #define LX_BOOTSOURCE_XSPI
 #include <../lx2160asi/lx2160a.rcwi>
-#include <../lx2160acex7/include/pll_2200_700_xxxx.rcwi>
+#include <../lx2160acex7/include/pll_2200_750_xxxx.rcwi>
 #include <../lx2160acex7/include/pll_xxxx_xxx_2900.rcwi>
 #include <../lx2160acex7/include/common.rcwi>
 #include <../lx2160acex7/include/SD1_8.rcwi>
diff --git a/lx2160acex7_rev2/clearfog-cx/rcw_2200_700_3200_8_5_2_auto.rcw b/lx2160acex7_rev2/clearfog-cx/rcw_2200_750_3200_8_5_2_auto.rcw
similarity index 91%
rename from lx2160acex7_rev2/clearfog-cx/rcw_2200_700_3200_8_5_2_auto.rcw
rename to lx2160acex7_rev2/clearfog-cx/rcw_2200_750_3200_8_5_2_auto.rcw
index e99a12b..046adc8 100644
--- a/lx2160acex7_rev2/clearfog-cx/rcw_2200_700_3200_8_5_2_auto.rcw
+++ b/lx2160acex7_rev2/clearfog-cx/rcw_2200_750_3200_8_5_2_auto.rcw
@@ -13,7 +13,7 @@
 
 #define LX_SR 2
 #include <../lx2160asi/lx2160a.rcwi>
-#include <../lx2160acex7/include/pll_2200_700_xxxx.rcwi>
+#include <../lx2160acex7/include/pll_2200_750_xxxx.rcwi>
 #include <../lx2160acex7/include/pll_xxxx_xxx_3200.rcwi>
 #include <../lx2160acex7/include/common.rcwi>
 #include <../lx2160acex7/include/SD1_8.rcwi>
diff --git a/lx2160acex7_rev2/clearfog-cx/rcw_2200_700_3200_8_5_2_sdhc.rcw b/lx2160acex7_rev2/clearfog-cx/rcw_2200_750_3200_8_5_2_sdhc.rcw
similarity index 91%
rename from lx2160acex7_rev2/clearfog-cx/rcw_2200_700_3200_8_5_2_sdhc.rcw
rename to lx2160acex7_rev2/clearfog-cx/rcw_2200_750_3200_8_5_2_sdhc.rcw
index dbc0a91..9798b74 100644
--- a/lx2160acex7_rev2/clearfog-cx/rcw_2200_700_3200_8_5_2_sdhc.rcw
+++ b/lx2160acex7_rev2/clearfog-cx/rcw_2200_750_3200_8_5_2_sdhc.rcw
@@ -15,7 +15,7 @@
 #define LX_SR 2
 #define LX_BOOTSOURCE_SDHC
 #include <../lx2160asi/lx2160a.rcwi>
-#include <../lx2160acex7/include/pll_2200_700_xxxx.rcwi>
+#include <../lx2160acex7/include/pll_2200_750_xxxx.rcwi>
 #include <../lx2160acex7/include/pll_xxxx_xxx_3200.rcwi>
 #include <../lx2160acex7/include/common.rcwi>
 #include <../lx2160acex7/include/SD1_8.rcwi>
diff --git a/lx2160acex7_rev2/clearfog-cx/rcw_2200_700_3200_8_5_2_xspi.rcw b/lx2160acex7_rev2/clearfog-cx/rcw_2200_750_3200_8_5_2_xspi.rcw
similarity index 91%
rename from lx2160acex7_rev2/clearfog-cx/rcw_2200_700_3200_8_5_2_xspi.rcw
rename to lx2160acex7_rev2/clearfog-cx/rcw_2200_750_3200_8_5_2_xspi.rcw
index c2ff13a..6b875e7 100644
--- a/lx2160acex7_rev2/clearfog-cx/rcw_2200_700_3200_8_5_2_xspi.rcw
+++ b/lx2160acex7_rev2/clearfog-cx/rcw_2200_750_3200_8_5_2_xspi.rcw
@@ -15,7 +15,7 @@
 #define LX_SR 2
 #define LX_BOOTSOURCE_XSPI
 #include <../lx2160asi/lx2160a.rcwi>
-#include <../lx2160acex7/include/pll_2200_700_xxxx.rcwi>
+#include <../lx2160acex7/include/pll_2200_750_xxxx.rcwi>
 #include <../lx2160acex7/include/pll_xxxx_xxx_3200.rcwi>
 #include <../lx2160acex7/include/common.rcwi>
 #include <../lx2160acex7/include/SD1_8.rcwi>
-- 
2.43.0

